src/VisualStudio/Core/Def/SymbolSearch/SymbolSearchService.cs

```diff
@@ -36,7 +36,8 @@ internal partial class SymbolSearchService :
         ISymbolSearchService,
         IDisposable
     {
-        private ConcurrentDictionary<string, AddReferenceDatabase> _sourceToDatabase = new ConcurrentDictionary<string, AddReferenceDatabase>();
+        // Value is typed as 'object' so we don't load the elfie dll until actually necessary.
+        private ConcurrentDictionary<string, object> _sourceToDatabase = new ConcurrentDictionary<string, object>();
```

Comment:
I'm not a fan of this bit.  Suggestions welcome on how i could keep things more strongly typed, while not causing anything in the typesystme to cause Elfie to load.